src/web/modules/transactions/update/update.http.ts

```diff
@@ -119,6 +130,9 @@ const validateAndAddDefaults = async function validateAndAddDefaults(csv: string
         if (!row.reason) {
           return cb(null, false, 'reason is missing')
         }
+        if (!row.admin_github_id) {
```

Comment:
would we ever want to allow this to be specified in the CSV?

Reply:
🤔 maybe not https://github.com/alphagov/pay-toolbox/pull/370/commits/1b9f740c23d7af96e57b86971c2e19a708411846